Documentation/network/kubernetes/ipam-multi-pool.rst

```diff
@@ -179,6 +186,8 @@ Validate installation
        [...]
        ✅ All 42 tests (295 actions) successful, 13 tests skipped, 0 scenarios skipped.
 
+  **Note:** The connectivity test requires a cluster with at least 2 worker nodes to complete successfully.
```

Comment:
Why is this required? I see that the example above tests from one node to another, but wouldn't it also work between local pods..?

Reply:
This is a general requirement of the connectivity tests, we should perhaps just be a warning when running the tests so we don't have to have similar comments everywhere.